objects/obj_creation_popup/main.js

```
@@ -0,0 +1,719 @@
+  );
+}
+
+function Toggle({ label, value, onChange }) {
+  return (
+    <label className="flex items-center gap-2 select-none cursor-pointer">
+      <span className="text-sm text-gray-600">{label}</span>
+      <span
+        onClick={()=>onChange(!value)}
+        className={`w-12 h-7 rounded-full p-1 transition ${value ? 'bg-blue-600' : 'bg-gray-300'}`}
+      >
+        <span className={`block w-5 h-5 bg-white rounded-full transition ${value ? 'translate-x-5' : ''}`}></span>
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)